File: team/views/pptuh/reduce.js
function(keys,values,rereduce) {
  if(!rereduce) {
    pts = 0
    tuh = 0
    for (v in values) {
      pts += values[v][0]
      tuh += values[v][1]
    }
    return [pts/tuh,pts,tuh]
  } else {
    pts = 0
    tuh = 0
    for(v in values) {
      pts += values[v][1]
      tuh += values[v][2]
    }
    return [pts/tuh,pts,tuh]
  }
}

File: stats/views/individual/reduce.cs
(keys,values,rereduce) -> 
  statline = {
    gp:0, tens:0, negs:0, tuh:0, pts:0
  }
  
  for value in values
    for stat of statline
      statline[stat] += value[stat]
  
  # ppg
  statline.ppg = statline.pts / statline.gp
  
  # ppth
  statline.ppth = statline.pts / statline.tuh
  
  statline